doc/meson.build

```diff
@@ -0,0 +1,81 @@
+# SPDX-License-Identifier: LGPL-2.1-or-later
+
+if want_mans == true
```

Comment:
`if` expects a boolean, and `want_mans == true` is a boolean expression that evaluates to `true == true` -> `true`.

So you can also just do: `if want_mans`